gamemode-zig/gamemode.zig

```diff
@@ -0,0 +1,76 @@
+//! Note: All the above requests can be blocking - dbus requests can and will block while the daemon
+//! handles the request. It is not recommended to make these calls in performance critical code
+
+pub const c = @import("gamemode_client.zig");
```

Comment:
what's the motivation for using translate-c vs. just directly importing the header?

Reply:
`zig translate-c` produced a zig file with an error that made it not compile. It tried to `@alignCast(std.meta.alignment(?*anyopaque), &REAL_internal_gamemode_request_start)`, for all the functions. In fetch_upstream.sh I delete that part to make it compile

Reply:
Ahh got it, makes sense